students/mayc4t/lesson06/mailroom4.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
+# #############################################################################
+# Written by: Mayc4t
+# Date: 4/27/2018
+# FileID: mailroom4.py
+#
+# Purpose: use unittest
+#Add a full suite of unit tests.
+
+#“Full suite” means all the code is tested. In practice, it’s very hard to test the user interaction, but you can test everything else. Make sure that there is as little untested code in the user interaction portion of the program as possible – hardly any logic.
+
+#This is a big step – you may find that your code is hard to test. If that’s the case, it’s a good sign that you should refactor your code.
+#
+# #############################################################################
+
+
+import collections
+from collections import *
+
+names = ["Lily Maycat", "Lulu Lemon", "Marc Jacobs", "Bobbi Brown", "Kate Spade"]
+gifts   = [i*[5000] for i in range(1,6)]
+
+def init_db():
+    global donors_db
+    donors_db = dict(zip(names, gifts))
```

Comment:
I would have this return the db rather than declaring a global inside the function